workers/unity/Packages/io.improbable.gdk.querybasedinteresthelper/InterestQuery.cs

```diff
@@ -47,15 +48,16 @@ public static InterestQuery Query(Constraint constraint)
         /// <param name="frequencyHz">
         ///     The maximum frequency (Hz) to return query results.
```

Comment:
Add a remark that `frequencyHz = 0` means there's no limit? I wouldn't assume that would be the case